installation/install/agent/windows.md

```diff
@@ -61,6 +61,7 @@ C:\> go-agent-16.1.0-1234-setup.exe /S /SERVERURL="https://10.12.20.47:8154/go"
     wrapper.java.additional.16=-Dcruise.config.foo=bar
     wrapper.java.additional.17=-Dcruise.config.other=baz
     ```
+    **Please note** : If the go-agent as an application is run by a local user account, then this user needs to have these required permissions to the go-agent folder, i.e. modify, read and execute, list folder contents and read permissions.
```

Comment:
Sorry, this seemed a little confusing to me. You meant to say that the regular user must have the right permissions, that is the permissions that the local system account has to access the go-agent directory, right?